.github/workflows/tests.yml

```diff
@@ -396,3 +325,231 @@ jobs:
       - run: echo "::add-matcher::.github/matchers/rust.json"
       - run: make slim-builds
       - run: make test-integration-splunk
+
+  # TODO: Upgrade to test once the tz issues are solved.
+  # https://github.com/timberio/vector/issues/3771
+  build-x86_64-unknown-linux-gnu:
+    name: Build - x86_64-unknown-linux-gnu
+    runs-on: ubuntu-20.04
+    if: |
+      !contains(github.event.pull_request.labels.*.name, 'ci-condition: skip')
+    steps:
+      - uses: actions/checkout@v2
+      - run: make ci-sweep
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - run: sudo bash scripts/environment/bootstrap-ubuntu-20.04.sh
+      - run: bash scripts/environment/prepare.sh
+      - run: echo "::add-matcher::.github/matchers/rust.json"
+      - run: make slim-builds
+      - run: make build-x86_64-unknown-linux-gnu
+
+  # TODO: Upgrade to test once the tz issues are solved.
+  # https://github.com/timberio/vector/issues/3771
+  build-aarch64-unknown-linux-gnu:
+    name: Build - aarch64-unknown-linux-gnu
+    runs-on: ubuntu-20.04
+    if: |
+      !contains(github.event.pull_request.labels.*.name, 'ci-condition: skip')
+    steps:
+      - uses: actions/checkout@v2
+      - run: make ci-sweep
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - run: sudo bash scripts/environment/bootstrap-ubuntu-20.04.sh
+      - run: bash scripts/environment/prepare.sh
+      - run: echo "::add-matcher::.github/matchers/rust.json"
+      - run: make slim-builds
+      - run: make build-aarch64-unknown-linux-gnu
+
+  build-deb-package:
+    name: Build .deb package
+    runs-on: ubuntu-latest
+    if: |
+      !contains(github.event.pull_request.labels.*.name, 'ci-condition: skip')
```

Comment:
How does this work with bors?

Reply:
Correct - this wouldn't impact bors.

Reply:
Interesting, why do we then need these `if`s? If I understand correctly, those jobs will only be run by bors now, but if bors doesn't propagate them - what's the point of having such conditions in the first place?

Reply:
I didn't remove them because it had no impact on testing bors or not.